tensorflow_datasets/core/dataset_builder.py

```diff
@@ -81,26 +102,26 @@ def name(self):
     return self._name
 
   @property
-  def version(self):
+  def version(self) -> str:
     return self._version
 
   @property
-  def supported_versions(self):
+  def supported_versions(self) -> List[str]:
     return self._supported_versions
 
   @property
-  def description(self):
+  def description(self) -> str:
     return self._description
 
-  def __repr__(self):
+  def __repr__(self) -> str:
     return "<{cls_name} name={name}, version={version}>".format(
         cls_name=type(self).__name__,
         name=self.name,
         version=self.version or "None")
 
 
 @six.add_metaclass(registered.RegisteredDataset)
-class DatasetBuilder(object):
+class DatasetBuilder(metaclass=abc.ABCMeta):
```

Comment:
`abc.ABCMeta` is automatically added by `registered.RegisteredDataset`. I don't think you can add two metaclasses for the same class.

Reply:
Done